src/xdb_transform.erl

```diff
@@ -259,7 +259,10 @@ repo_sup_spec(Repo) ->
 repo_fun_template(Mod, Fun, Arity, Repo, Adapter) ->
   Args = splicing_args(Arity),
   Body = build_repo_fun(Repo, Adapter, Mod, atom_to_list(Fun), Args),
-  {Fun, Arity, Body}.
+  case build_repo_fun_spec(Repo, Fun, Args) of
```

Comment:
The thing is, when you have behaviour and the callbacks are well defined in terms of specs, it shouldn't be necessary to duplicate the same spec every time you implement that behaviour, that's why you can use the `@hidden` tag. I think it is something that has to be improved for dialyzer, but in the meantime, I will disable mandatory specs for all exported functions.

Reply:
I get your point, somehow having specs on callback is not enough. How would you disable mandatory specs?